resources.tf

```diff
@@ -40,6 +40,11 @@ resource "aws_s3_bucket" "published" {
     enabled = true
   }
 
+  logging {
+    target_bucket = data.terraform_remote_state.security-tools.outputs.logstore_bucket.id
+    target_prefix = "S3Logs/${random_id.published_bucket.hex}-ci/ServerLogs"
```

Comment:
This doesn't need the `-ci` suffix on the hex. The bucket name is the hex value.

Reply:
Amended the code